VisualizationBase/VisualizationBase.pro

```diff
@@ -128,7 +128,9 @@ HEADERS += src/overlays/IconOverlayStyle.h \
     src/declarative/DynamicGridFormElement.h \
     src/overlays/MessageOverlay.h \
     src/overlays/MessageOverlayStyle.h \
-    src/items/ViewItem.h
+    src/items/ViewItem.h \
+    src/items/VNodeReference.h \
```

Comment:
We should have one visualization VViewItemNode that uses the Declarative visualization mechanism and declares multiple forms (for now, one form for spares, and one form for normal node referencing).